content/pt/docs/languages/go/exporters.md

```diff
@@ -2,13 +2,12 @@
 title: Exporters
 aliases: [exporting_data]
 weight: 50
-default_lang_commit: 07431d6e22a33faa6775f2c1f40aa122990dc214
-drifted_from_default: true
+default_lang_commit: 351727ae36f706eb80583ada2b589de263aa72c2
 # prettier-ignore
-cSpell:ignore: otlplog otlploggrpc otlploghttp otlpmetric otlpmetricgrpc otlpmetrichttp otlptrace otlptracegrpc otlptracehttp promhttp stdoutlog stdouttrace
+cSpell:ignore: otlplog otlploggrpc otlploghttp otlpmetric otlpmetricgrpc otlpmetrichttp otlptrace otlptracegrpc otlptracehttp stdoutlog stdouttrace
 ---
 
-{{% pt/docs/languages/exporters/intro go %}}
+{{% pt/docs/languages/exporters/intro %}}
```

Comment:
Oh no, I missed to notice that this doesn't have the most important change: there should be no `pt/` prefix on that shortcode path. I'll create an issue for this.

Reply:
See:

- #6812